src/main/java/edu/unc/lib/boxc/migration/cdm/services/PostMigrationReportVerifier.java

```diff
@@ -42,18 +54,28 @@ public VerificationOutcome verify() throws IOException {
         ) {
             long currentNum = 0;
             updateProgressDisplay(currentNum, totalRecords);
+            var baseUrl = chompbConfig.getBxcEnvironments().get(project.getProjectProperties().getBxcEnvironmentId()).getHttpBaseUrl();
+            bxcRecordBaseUrl = baseUrl + RECORD_PATH;
+            bxcApiBaseUrl = baseUrl + API_PATH;
```

Comment:
We should generally avoid setting instance variables as a side effect in a method that could be called more than once since it can lead to unexpected interference. They're each only used in one place, so we could either pass them in or pass in the baseUrl and construct the path as needed.